src/charm.py

```diff
@@ -200,6 +200,8 @@ def _configure_nginx_ingress_controller(self):
                 "use_forwarded_headers": (
                     "true" if self.config["ingress-use-forwarded-headers"] else "false"
                 ),
+                # NOTE(Hue): The default comes from https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#proxy-real-ip-cidr
+                "proxy_real_ip_cidr": self.config.get("ingress-proxy-real-ip-cidr", "0.0.0.0/0"),
```

Comment:
Thanks for finding this in the docs. 

i also confirmed this default [here](https://github.com/kubernetes/ingress-nginx/blob/controller-v1.11.2/internal/ingress/controller/config/config.go#L776) in the go code for the controller on the version shipped in this charm, `1.11.2`